baselines/dom.generated.d.ts

```diff
@@ -13009,10 +13009,14 @@ interface ElementDefinitionOptions {
     extends: string;
 }
 
+interface ElementCreationOptions {
+    is: string;
```

Comment:
Fair point. I was going off of the listed algorithm, which handles the case where `is` isn't provided. Looking at chrome's implementation confirms your reading, it doesn't crash, but it does `setAttribute('is', '[object Object]')`.

In short, I withdraw the suggestion, `is` should be required.